<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaticDIClass</class>
 <widget class="QDialog" name="StaticDIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Static DI</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>-4</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>91</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#background{background-image:url(:/StaticDI/Resources/Background.png)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfig">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>398</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>52</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>361</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>361</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Port No.    Bit7                          4            3                             0          Hex</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>71</y>
     <width>370</width>
     <height>317</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>369</width>
     <height>317</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>370</width>
     <height>317</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>369</width>
     <height>317</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>0</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Time_Frame">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>230</y>
     <width>370</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_4_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>4.5V</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_neg_1_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>-1.5V</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
